packages/rancher-monitoring/overlay/charts/grafana/templates/nginx-config.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-nginx-proxy-config
+  namespace: {{ template "grafana.namespace" . }}
+  labels:
+    {{- include "grafana.labels" . | nindent 4 }}
+data:
+  nginx.conf: |-
```

Comment:
Copied straight from [Monitoring V1](https://github.com/rancher/system-charts/blob/d6f4e8f7056917ed013ddd14c9a33f750dc3e95d/charts/rancher-monitoring/v0.1.3/charts/grafana/templates/nginx-configmap.yaml#L45-L111). We don't need the scripts from the Monitoring V1 ConfigMap since we aren't trying to validate the requests coming into Prometheus / Grafana based on service account credentials (unlike Monitoring V1, which enforces RBAC on Prometheus via [`rancher/prometheus-auth`](https://github.com/rancher/prometheus-auth)